src/server/qgsserversettings.cpp

```diff
@@ -162,6 +162,27 @@ void QgsServerSettings::initSettings()
                                       };
   mSettings[ sShowGroupSeparator.envVar ] = sShowGroupSeparator;
 
+  // max height
+  const Setting sMaxHeight = { QgsServerSettingsEnv::QGIS_SERVER_WMS_MAX_HEIGHT,
+                               QgsServerSettingsEnv::DEFAULT_VALUE,
+                               "Maximum height for a WMS request. The most conservative between this and the project one is used",
```

Comment:
@m-kuhn why no `QStringLiteral` is required here? (and below)

Reply:
Hmmm... I didn't pay attention to that, it would probably be good?

Reply:
@m-kuhn I dit the same as in https://github.com/qgis/QGIS/blob/0bc43b6dc7f8a514b164277c133a138afdd04533/src/server/qgsserversettings.cpp#L124 and all other Settings
do you want me to add QStringLiteral to all of them?

Reply:
Changing this everywhere sounds good to me

Reply:
> Changing this everywhere sounds good to me

+1